frontend/src/types.ts

```diff
@@ -317,6 +317,9 @@ export type PodContainer = {
   livenessProbe?: Record<string, unknown>;
   readinessProbe?: Record<string, unknown>;
   volumeMounts?: VolumeMount[];
+  terminationMessagePath?: string;
+  terminationMessagePolicy?: string;
+  securityContext?: unknown;
```

Comment:
Additional types required by mocks otherwise typecheck fails. Used `unknown` for `securityContext` because it's a large data structure and we never actually inspect it. This is simply used by our mocks right now.